crates/biome_css_analyze/tests/specs/nursery/noDuplicateProperties/valid.css

```diff
@@ -15,6 +15,12 @@ a { color: pink; @media { color: orange; &::before { color: black; } } }
 
 a { --custom-property: 0; --custom-property: 1; }
 
+a {
+    @supports (color: pink) {
+        color: pink;
+    }
+}
+
```

Comment:
This looks good to me as a positive test, but I'd like to see a negative test case as well. FYI, you can add other files in this folder, you aren't necessarily limited to `valid.css` and `invalid.css`

Reply:
Thank you! I added more tests for both valid and invalid cases, and I confirmed that the `noDuplicateProperties` lint rule properly throws errors for invalid test cases. Please check it again!